CCDB/src/CcdbApi.cxx

```diff
@@ -138,7 +138,7 @@ void CcdbApi::init(std::string const& host)
     snapshotReport += ')';
   }
 
-  mNeedAlienToken = (host.find("https://") == 0) || (host.find("http://alice-ccdb.cern.ch") == 0);
+  mNeedAlienToken = (host.find("https://") == 0) || (host.find("alice-ccdb.cern.ch") == 0);
```

Comment:
OK, but then the `find` will not give 0 if the `http` is provided. Perhaps one should use `host.find("alice-ccdb.cern.ch") <= 7 ` ?

Reply:
Indeed, thx for spotting. I have changed it to `!= std::string::npos`, which should be even more save, e.g. if there is whitespace at the beginning.
I think random `https://` or `alice-ccdb.cern.ch` somewhere else in the string is pretty unlikely.